qiskit/pulse/program.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The pulse program. A collection of pulses schedules."""
+
+import copy
+from typing import List, Optional
+
+from qiskit.pulse.schedule import Schedule
+
+
+class Program:
```

Comment:
This is `List[Experiments]` not a single program?

Reply:
Yes, this allows us to compile the `Program` as a whole (which is really an execution on the device). Perhaps this would be better titled a `Module`. This is necessary for cases where we wish to align measurements to occur at the same time or concepts like global barriers. Perhaps a better name would be `Module`.